packages/types/src/payment-types.ts

```diff
@@ -51,17 +46,27 @@ export interface IPaymentNetworkCreateParameters<T = any> {
 
 export type PaymentNetworkCreateParameters =
```

Comment:
the "complexity" of the type prevented usage of [Extract](https://www.typescriptlang.org/docs/handbook/utility-types.html#extracttype-union)

Reply:
Arg, I remembered struggling with this.